pkg/glusterutils/exporterd.go

```diff
@@ -145,3 +148,45 @@ func ExecuteCmd(cmd string) ([]byte, error) {
 	out, err := exec.Command(cmdstr, args...).Output()
 	return out, err
 }
+
+// EnableVolumeProfiling enables profiling for volumes
+func EnableVolumeProfiling(gluster GInterface, config *Config) error {
```

Comment:
this works only for gd2?

Reply:
Yeah. Working out the logic for gd1 case and would update the PR.